osh/hub/static-assets/css/redhat.css

```diff
@@ -793,3 +793,14 @@ div#defects div.events_list div.code {
 }
 
 /**/
+
+
+/**/
```

Comment:
Why do you insert the empty comments and the doubled empty lines?

Reply:
Moved the changes to another block.